pkg/nodeca/nodeca.go

```
@@ -0,0 +1,181 @@
+
+// copyFrom copies all files from srcdir directory to dstdir, creating a
+// subdirectory per file with transformed name and copying it as ca.crt.
+// For example, srcdir/registry..5000 file is copied to
+// dst/registry:5000/ca.crt. Errors copying individual files are logged
+// but not returned. If dstdir does not exist this function creates it.
+func copyFrom(srcdir, dstdir string) error {
+	if dststat, err := os.Stat(dstdir); err != nil {
+		if !errors.Is(err, os.ErrNotExist) {
+			return fmt.Errorf("reading destination: %w", err)
+		}
+		if err := os.MkdirAll(dstdir, 0755); err != nil {
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofumpt)